README.md

```diff
@@ -76,153 +35,6 @@ For example, in bash, adding something along the lines of:
-  done:                Transition issue to Done state
-  dup:                 Mark issues as duplicate
-  edit:                Edit issue details
-  editmeta:            View 'edit' metadata
-  epic add:            Add issues to Epic
-  epic create:         Create Epic
-  epic list:           Prints list of issues for an epic with optional search criteria
-  epic remove:         Remove issues from Epic
-  export-templates:    Export templates for customizations
-  fields:              Prints all fields, both System and Custom
-  in-progress:         Transition issue to Progress state
-  issuelink:           Link two issues
-  issuelinktypes:      Show the issue link types
-  issuetypes:          Show issue types for a project
-  labels add:          Add labels to an issue
-  labels remove:       Remove labels from an issue
-  labels set:          Set labels on an issue
-  list:                Prints list of issues for given search criteria
-  login:               Attempt to login into jira server
-  logout:              Deactivate session with Jira server
-  rank:                Mark issues as blocker
-  reopen:              Transition issue to reopen state
-  request:             Open issue in requestr
-  resolve:             Transition issue to resolve state
-  start:               Transition issue to start state
-  stop:                Transition issue to stop state
-  subtask:             Subtask issue
-  take:                Assign issue to yourself
-  todo:                Transition issue to To Do state
-  transition:          Transition issue to given state
-  transitions:         List valid issue transitions
-  transmeta:           List valid issue transitions
-  unassign:            Unassign an issue
-  unexport-templates:  Remove unmodified exported templates
-  view:                Prints issue details
-  vote:                Vote up/down an issue
-  watch:               Add/Remove watcher to issue
-  worklog add:         Add a worklog to an issue
-  worklog list:        Prints the worklog data for given issue
-  session:             Attempt to login into jira server
-
-```
-
-## v1 vs v0 changes
-
-###### **Golang library import**
-For the new version of go-jira you should use:
-```
-import "gopkg.in/Netflix-Skunkworks/go-jira.v1"
-```
-
-If you have code that depends on the old apis, you can still use them with this import:
-```
-import "gopkg.in/Netflix-Skunkworks/go-jira.v0"
-```
-
-###### **Configs per command**
-Instead of requiring a executable template to get configs for a given command now you can create a config to be applied to a command.  So if you want to use `template: table` by default for your `jira list` you can now do:
-```
-$ cat $HOME/.jira.d/list.yml
-template: table
-```
-Where previously you needed something like:
-```
-# cat $HOME/.jira.d/config.yml
-#!/bin/sh
-case $JIRA_OPERATION in
-    list)
-      echo "template: table";;
-esac
-```
-
-###### **Custom Commands**
```

Comment:
Personally, I'd like to keep this section. It was super helpful to me.

Reply:
Not sure I follow - there's still the regular custom commands section. This is the section that covers how the behavior changed between v0 and v1. Like the commit says, v0 is ancient, so I don't think this is relevant anymore.

Reply:
Oh. Then let's remove it!